cmd/common.go

```diff
@@ -61,6 +61,9 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 	if err != nil {
 		return err
 	}
+	if dumpConfig.SkipConsumers {
```

Comment:
I am wondering if we can figure a way to get rid of these global mutable vars?

Reply:
Not a big deal and certainly not in the scope of this PR.